src/components/builder.ts

```diff
@@ -270,7 +270,7 @@ export class Builder implements IBuilder {
                 command += ' ' + args[0]
             }
             this.extension.logger.addLogMessage(`cwd: ${path.dirname(rootFile)}`)
-            this.currentProcess = cs.spawn(command, [], {cwd: path.dirname(rootFile), env: envVars, shell: true})
+            this.currentProcess = cs.spawn(command, [], {cwd: path.dirname(rootFile), env: envVars, shell: getShell()})
```

Comment:
The function `getShell` returns `false` if no specific shell is specified. it changes the default behaviour.

The `command` variables holds the complete command + its arguments, so we must pass `shell: true` to make it work;

Reply:
Exactly same time! Fixed in the new commit. May need to add a small note here.